lib/virtfs/ext3/superblock.rb

```diff
@@ -1,19 +1,16 @@
 # encoding: US-ASCII
 
-require 'fs/ext3/group_descriptor_table'
-require 'fs/ext3/inode'
+require_relative 'group_descriptor_table'
+require_relative 'inode'
 
 require 'binary_struct'
-require 'util/miq-uuid'
+require 'virt_disk/disk_uuid'
```

Comment:
This file doesn't exist, right?  I noticed that you added a dependency to the UUIDTools gem in the Gemfile - is your intent to change this to use that other gem?